packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -250,8 +250,9 @@ export const report = (data: AnalyticsEvent, force = false) => async (
     _dispatch: Dispatch,
     getState: GetState,
 ) => {
-    if (isDev()) {
-        // on dev, do nothing
+    // @ts-ignore
+    if (window.location.hostname === 'localhost') {
```

Comment:
this file is not furture proof, or react-native proof. Would be nice to get rid of window references here.